jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/awsrules/cloudtrail/EnableCloudTrailCloudwatchLogsIntegration.java

```diff
@@ -0,0 +1,63 @@
+package com.tmobile.cloud.awsrules.cloudtrail;
+import com.nimbusds.oauth2.sdk.util.MapUtils;
+import com.tmobile.cloud.awsrules.utils.PacmanUtils;
+import com.tmobile.cloud.constants.PacmanRuleConstants;
+import com.tmobile.pacman.commons.PacmanSdkConstants;
+import com.tmobile.pacman.commons.exception.InvalidInputException;
+import com.tmobile.pacman.commons.policy.Annotation;
+import com.tmobile.pacman.commons.policy.BasePolicy;
+import com.tmobile.pacman.commons.policy.PacmanPolicy;
+import com.tmobile.pacman.commons.policy.PolicyResult;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+import java.util.*;
+
+@PacmanPolicy(key = "enable-cloudtrail-cloudwatch-logs-integration", desc = "This rule checks that CloudTrail trails are integrated with CloudWatch Logs", severity = "critical", category = PacmanSdkConstants.SECURITY)
+public class EnableCloudTrailCloudwatchLogsIntegration extends BasePolicy {
+
+    private static final Logger logger = LoggerFactory.getLogger(EnableCloudTrailCloudwatchLogsIntegration.class);
+
+    @Override
+    public PolicyResult execute(final Map<String, String> ruleParam, Map<String, String> resourceAttributes) {
+
+        logger.debug("========EnableCloudTrailCloudwatchLogsIntegration started=========");
+        Annotation annotation = null;
+        MDC.put("executionId", ruleParam.get("executionId"));
+        MDC.put("ruleId", ruleParam.get(PacmanSdkConstants.POLICY_ID));
+
+        String severity = ruleParam.get(PacmanRuleConstants.SEVERITY);
+        String category = ruleParam.get(PacmanRuleConstants.CATEGORY);
+
+        String cloudWatchLogsLogGroupArn = resourceAttributes.get(PacmanRuleConstants.CLOUD_WATCH_LOGS_ARN);
+
+        List<LinkedHashMap<String, Object>> issueList = new ArrayList<>();
+        LinkedHashMap<String, Object> issue = new LinkedHashMap<>();
+
+        if (MapUtils.isNotEmpty(ruleParam) && Boolean.TRUE.equals(!PacmanUtils.doesAllHaveValue(severity, category))) {
```

Comment:
Why do you need `Boolean.TRUE.equals`?

Reply:
SonarLint is suggesting to use the primitive boolean expression here because Boxed "Boolean" should be avoided in boolean expressions

Reply:
I don't remember when started using SonarLint as source of truth...
What value type this returns?
`!PacmanUtils.doesAllHaveValue(severity, category)`
For it looks like boolean.
Do you personally (not SonarLint) have doubts that if you remove `Boolean.TRUE.equals` then all will fail?